sdk/ai/azure-ai-assistants/tests/model_assistants_test_base.py

```diff
@@ -0,0 +1,493 @@
+    #                     "type": "string",
+    #                     "description": "Second argument description",
+    #                 },
+    #             },
+    #             "required": ["first_argument", "second_argument"],
+    #         },
+    #     )
+    # )
+
+    # TOOL2 = sdk.models.ChatCompletionsToolDefinition(
+    #     function=sdk.models.FunctionDefinition(
+    #         name="my-second-function-name",
+    #         description="My second function description",
+    #         parameters={
+    #             "type": "object",
+    #             "properties": {
+    #                 "first_argument": {
+    #                     "type": "int",
+    #                     "description": "First argument description",
+    #                 },
+    #             },
+    #             "required": ["first_argument"],
+    #         },
+    #     )
+    # )
+
+    # TODO sophia update this too i think? 
+    # Expected JSON request payload in regression tests. These are common to
+    # sync and async tests, therefore they are defined here.
+    CHAT_COMPLETIONS_JSON_REQUEST_PAYLOAD = '{"messages": [{"role": "system", "content": "system prompt"}, {"role": "user", "content": "user prompt 1"}, {"role": "assistant", "tool_calls": [{"function": {"name": "my-first-function-name", "arguments": {"first_argument": "value1", "second_argument": "value2"}}, "id": "some-id", "type": "function"}, {"function": {"name": "my-second-function-name", "arguments": {"first_argument": "value1"}}, "id": "some-other-id", "type": "function"}]}, {"role": "tool", "tool_call_id": "some id", "content": "function response"}, {"role": "assistant", "content": "assistant prompt"}, {"role": "user", "content": [{"type": "text", "text": "user prompt 2"}, {"type": "image_url", "image_url": {"url": "https://does.not.exit/image.png", "detail": "high"}}]}], "stream": true, "frequency_penalty": 0.123, "max_tokens": 321, "model": "some-model-id", "presence_penalty": 4.567, "response_format": {"type": "json_object"}, "seed": 654, "stop": ["stop1", "stop2"], "temperature": 8.976, "tool_choice": "auto", "tools": [{"function": {"name": "my-first-function-name", "description": "My first function description", "parameters": {"type": "object", "properties": {"first_argument": {"type": "string", "description": "First argument description"}, "second_argument": {"type": "string", "description": "Second argument description"}}, "required": ["first_argument", "second_argument"]}}, "type": "function"}, {"function": {"name": "my-second-function-name", "description": "My second function description", "parameters": {"type": "object", "properties": {"first_argument": {"type": "int", "description": "First argument description"}}, "required": ["first_argument"]}}, "type": "function"}], "top_p": 9.876, "key1": 1, "key2": true, "key3": "Some value", "key4": [1, 2, 3], "key5": {"key6": 2, "key7": false, "key8": "Some other value", "key9": [4, 5, 6, 7]}}'

```

Comment:
Assistants have multiple JSON payload requests (for different operations) we may want to use in tests, but I am not sure if we want to copy the pattern.